src/python/pants/backend/docker/subsystems/dockerfile_parser_test.py

```diff
@@ -73,12 +76,19 @@ def test_parsed_injectables(files: list[tuple[str, str]], rule_runner: RuleRunne
     addr = Address("test")
     info = rule_runner.request(DockerfileInfo, [DockerfileInfoRequest(addr)])
     assert info.from_image_build_args.to_dict() == {"BASE_IMAGE": ":base"}
+    assert info.copy_build_args.to_dict() == {
+        "PEX_BIN": ":hello",
+        "PEX_BIN_DOTTED_PATH": "dotted.path.as.arg/dpaa.pex",
+    }
     assert info.copy_source_paths == (
         "some.target/binary.pex",
         "some.target/tool.pex",
         "binary",
         "another/cli.pex",
         "tool",
+        "${PEX_BIN}",  # TODO: Should this be here?
+        "${PEX_BIN_DOTTED_PATH}",  # TODO: Should this be here?
```

Comment:
I think either "no", _or_ as their respective expanded value (i.e. the build arg value that will be used.)

Reply:
Turn out replacing it with the build arg that should be used isn't possible in the parser. ARGs can be supplied in a variety of ways outside of the parser (target fields, global config). Also they will definitely not be useful as source paths (I mean, someone _could_ call a file `${PEX_BIN}`, but they deserve it). So it makes sense to just leave them out (which is pretty easy)